modules/channel-web/src/views/lite/components/messages/Message.tsx

```diff
@@ -99,6 +99,15 @@ class Message extends Component<Renderer.Message> {
     return '*Unsupported message type*'
   }
 
+  handleContextMenu = e => {
+    const showContextMenu = window.botpress.extensions && window.botpress.extensions.showContextMenu
+    if (showContextMenu) {
+      const customActions = this.props.store.view.customActions
+      const id = this.props.incomingEventId
+      showContextMenu(e, { id, customActions })
```

Comment:
What do you think of passing the complete props to showContextMenu? In this case we use incomingEventId, but it may be possible that we need another of the available prop